### PR TITLE
translate local and templates to Punjabi(pu)

### PR DESCRIPTION
**Pull request by naman360**

### CODE REVIEW

**Comment by spielers** [comment]
@naman360  Looks great !

**Comment by naman360** [comment]
@spielers @eldadfux Review?

**Comment by spielers** [comment]
Hey, @naman360  Thanks for your contribution as I saw You have stated (pu) Punjabi but according to ISO 639-1 Punjabi is denoted by (pa). So I suggest you update it with the correct notation.

best,
spielers

**Comment by naman360** [comment]
@spielers I am done with changes, have a look?

File: app/config/locale/translations/pa.countries.php [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    'JP' => 'ਜਪਾਨ',
+    'KZ' => 'ਕਜ਼ਾਕਿਸਤਾਨ',
+    'KE' => 'ਕੀਨੀਆ',
+    'KG' => 'ਕਿਰਗਿਸਤਾਨ',
+    'KH' => 'ਕੰਬੋਡੀਆ',
+    'KI' => 'ਕਿਰੀਬਾਤੀ',
+    'KN' => 'ਸੈਂਟ ਕਿੱਟਸ ਅਤੇ ਨੇਵਿਸ',
+    'KR' => 'ਦੱਖਣੀ ਕੋਰੀਆ',
+    'KW' => 'ਕੁਵੈਤ',
+
```
Comment (eldadfux) [inline app/config/locale/translations/pa.countries.php:102]: Please remove the extra space

File: app/config/locale/translations/pa.countries.php [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    'PA' => 'ਪਨਾਮਾ',
+    'PE' => 'ਪੇਰੂ',
+    'PH' => 'ਫਿਲਪੀਨਜ਼',
+    'PW' => 'ਪਲਾਉ',
+    'PG' => 'ਪਪੁਆ ਨਵਾਂ ਗਿੰਨੀ',
+    'PL' => 'ਪੋਲੈਂਡ',
+    'KP' => 'ਉੱਤਰੀ ਕੋਰਿਆ',
+    'PT' => 'ਪੋਰਟੁਗਲ',
+    'PY' => 'ਪੈਰਾਗੁਏ',
+
```
Comment (eldadfux) [inline app/config/locale/translations/pa.countries.php:152]: Please remove the extra space

File: app/config/locale/translations/pa.countries.php (hunk 1 above)
Comment (naman360) [reply app/config/locale/translations/pa.countries.php:102]: okay

**Comment by naman360** [comment]
@eldadfux done with changes, have a look

**Review by Kolkies — APPROVED** [review]

**Review by spielers — APPROVED** [review]
Thanks for your contribution @naman360,  We will get this merged.

**Comment by naman360** [comment]
Thanks!

**Comment by spielers** [comment]
Hey, @naman360 Thanks for your contribution will get you merged

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like me to send you some cool Appwrite stickers and magnets! Please note that shipping might take longer times because of COVID19.

As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

**Pull request merged**